### ISSUE TITLE
[Bug]: Discord Invitation Expired

### ISSUE THREAD

**Issue by HanifNaOH** [virtual-puppet-project/vpuppr#226]
### Description

![image](https://github.com/virtual-puppet-project/vpuppr/assets/86117958/4509c6a8-462e-4d12-80f1-75803cd86864)

### Steps to reproduce

1. Go for Discord invitation at collab.md or at Issues Report

### App version

-

### Operating system

Linux

**Comment by Lualttt** [comment]
yeah, there also havent been any updates for 4 months and found a mastodon post about that the project is on an indefinite hiatus https://vt.social/@rogueren/111789040342541328 which lines up with the dates, but i cant find any proof or actual info

so it might just be abandoned and the discord could be deleted because of it
